src/c2pa/c2pa.py

```diff
@@ -205,6 +218,49 @@ class C2paSignerInfo(ctypes.Structure):
         ("ta_url", ctypes.c_char_p),
     ]
 
+    def __init__(self, alg=None, sign_cert=None, private_key=None, ta_url=None):
```

Comment:
This is a lot of syntactic sugar to make enums reusable and less dependent on underlying (native) types. Also Lots of type checkings checks when converting.